### ISSUE TITLE
[Feature request] Add Search Engines

### ISSUE THREAD

**Issue by hl2guide** [JasonStein/Notepads#1059]
**Is your feature request related to a problem? Please describe.**
It frustrates me that Qwant and Brave Search aren't available for the "Default Search Engine" setting.

**Describe the solution you'd like**
Please add radio items for Qwant and Brave Search.

**Describe alternatives you've considered**
Custom Search Engine field.

**Additional context**
N/A

**Comment by JasonStein** [comment]
<img width="1215" alt="Screen Shot 2022-03-16 at 10 40 01 AM" src="https://user-images.githubusercontent.com/1166162/158653365-6bcff88d-46dc-48a2-ae3a-f0fd64f08f9c.png">
<img width="342" alt="Screen Shot 2022-03-16 at 10 45 02 AM" src="https://user-images.githubusercontent.com/1166162/158654202-331b37ca-2093-4186-9dc1-0a6487d347ed.png">

**Comment by hl2guide** [comment]
Yes, I'm aware, it'd be nice to have more in-built options for novice users who don't understand `{0}`.

**Comment by JasonStein** [comment]
@EmirhakanTanhan Thx for the PR btw, but I am still not convinced yet:

1. The user base of Qwant and Brave is too low even compared to Duck Duck Go. Most of the Notepads users are using Google/Bing.
2. Having too many options in the settings is against Notepads' design and philosophy.
3. There is already an option for you to configure any arbitrary search engine and I am pretty sure for people using Qwant and Brave knows how to configure it (think about it).

**Comment by EmirhakanTanhan** [comment]
Hi @JasonStein , you probably mentioned me by mistake right?

**Comment by JasonStein** [comment]
> 

Oh I am so sorry:)

@Mennaruuk

**Comment by JasonStein** [comment]
> Hi @JasonStein , you probably mentioned me by mistake right?

Sorry about that!
